docs/releasenotes/service.md

```diff
@@ -10,14 +10,54 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 Service Release Notes for the current year are listed here. If you need to see previous Service Release Notes, check the [Archive](docs/releasenotes/archive/index.md).
 
 ---
-### September 12, 2022 (Alerts)
+
+## September 19, 2022 (Security)
+
+Update - We've improved out Access Key UI to make it easy to spot Access Keys that haven't been used lately. Now the Access Keys management page displays the date and time an Access Key was last used to make a request to an API endpoint. For more information, see [Access Keys](docs/manage/security/access-keys.md) .
+
+In a future update, Sumo Logic will, by default, automatically disable Access Keys that have not been used for 30 days. Your Sumo Logic admin will be able to configure the period of time before an unused key will be disabled.  
+
+---
+
+## September 15, 2022 (Apps)
+
+Update - GitHub Advanced Security dashboards are now available for the Sumo Logic App for GitHub. These new dashbaords include Code Scanning Alerts, Pushes, Secret Scanning Alerts, Security and analyses and Repository Vulnerability alerts.
+
+---
+
+## September 12, 2022 (Alerts)
 
 New - We’re happy to announce the release of Alert Grouping, which allows you to generate more than one alert from a given monitor by specifying a group condition on one or more fields. For example, rather than creating multiple monitors for each `service`, you could create one single monitor that notifies you when some metric (i.e., CPU utilization, error count) goes above the threshold for a given `service`. [Learn more](docs/alerts/monitors/alert-grouping.md).
 
 New - Configurable Resolution Window for Logs allows more quickly resolve alerts when the underlying issues are fixed. You can configure how long a monitor will wait, before resolving the alert, when the underlying issues was corrected (earlier the monitor waited one complete window before resolving). [Learn more](docs/alerts/monitors#Configurable-Resolution-Window-for-Logs).
 
 New - You can now access your monitor playbook as a template variable, `{{playbook}}`. You can reference this template variable to customize your notification payloads similar to any other template variable. [Learn more](docs/alerts/alert-variables.md).
 
+---
+
```

Comment:
You need to close the space between the triple-dotted lines and the header in order to preserve the formatting

Reply:
OK